error_check_logic.py

```diff
@@ -25,7 +33,38 @@ def setStatus(self, institution, session, status = 'UNKNOWN'):
 
   @classmethod
   def getStatus(self, institution, session):
```

Comment:
Oops never mind. This is a class method. :)

Usually we call the first parameter cls instead of self if it's a class method.